deployment/terraform/live/main.tf

```diff
@@ -29,25 +37,27 @@ module "identity_pool" {
 }
 
 module "docker_host" {
-  source              = "../modules/docker_host"
-  vpc_cidr_block      = "${var.vpc_cidr_block}"
-  private_subnet_id   = "${module.vpc.private_subnet_a_id}"
-  vpc_id              = "${module.vpc.vpc_id}"
-  security_group_name = "${terraform.workspace}-docker"
+  source            = "../modules/docker_host"
+  vpc_cidr_block    = "${var.vpc_cidr_block}"
+  private_subnet_id = "${module.vpc.private_subnet_a_id}"
+  vpc_id            = "${module.vpc.vpc_id}"
+  tags              = "${local.tags}"
 }
 
-
 module "vpc" {
   source               = "../modules/vpc"
   vpc_cidr_block       = "${var.vpc_cidr_block}"
   public_cidr_block    = "${var.public_cidr_block}"
   private_cidr_block_a = "${var.private_cidr_block_a}"
   private_cidr_block_b = "${var.private_cidr_block_b}"
-  availability_zone    = "${var.availability_zone_a}"
+  availability_zone_a  = "${var.availability_zone_a}"
+  availability_zone_b  = "${var.availability_zone_b}"
```

Comment:
Is there a correspondence between the cidr_blocks and the availability_zones? Would this be a good place for a comment to explain what's going on here, or that might go somewhere else?

Reply:
You might want to ask @scottx611x since this is his code :) I've simply removed an unused variable `availability_zone` and added variables that were [declared](https://github.com/refinery-platform/refinery-platform/blob/95025f0a8d1bbf2e90fbc4094928989c3e3279a6/deployment/terraform/live/vars.tf) but not assigned: `availability_zone_a` and `availability_zone_b`.

Reply:
The relationship between the private cidr blocks a & b and availability zones a & b is that to be able to put an RDS instance inside of a VPC there must be two subnets in two different Availability Zones in the region where we deploy. Since we don't have the RDS hotswapping enabled, `private_cidr_block_b` and `availability_zone_b` don't matter other than to satisfy that requirement. I thought I had some comment about this in the TF code let me check